README.md

```diff
@@ -284,6 +284,26 @@ func main() {
 }
 ```
 
+### Using pflag with go test
+`pflag` skips parsing of shorthand version of go test specific flags (i.e. the one passed as `-test.`)
+See issues: [#63](https://github.com/spf13/pflag/issues/63) and [#238](https://github.com/spf13/pflag/issues/238) for more details.
+
+**Example**: You want to separately parse go test flags anyway
+```go
+import (
+	goflag "flag"
+	flag "github.com/spf13/pflag"
+)
+
+var ip *int = flag.Int("flagname", 1234, "help message for flagname")
+
+func main() {
+	flag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+    ParseGoTestFlags(os.Args[1:], goflag.CommandLine)
+	flag.Parse()
+}
+```
+
```

Comment:
I think this example could be more informative. If I parse flags like this, what does that mean for usage? How do I run this program and pass flags to it, and in what ways is that different from the behavior when you don't call `ParseGoTestFlags`? What values will I get for various flags in this case?